crm_department/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM with Department Categorization
+==================================
+
+
+
+Add the department on Sales Team (with related field on Lead &amp; Opportunities)
+as well as the related filter and button in the search form.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+*(in alphabetical order)*
+
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Ángel Moya <angel.moya@domatix.com>
+* Charbel Jacquin <charbel.jacquin@camptocamp.com>
+* Daniel Reis <dgreis@sapo.pt>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Leonardo Pistone <leonardo.pistone@camptocamp.com>
+* Omar (Pexego) <omarcs7r@gmail.com>
+* Pedro M. Baeza <pedro.baeza@gmail.com>
```

Comment:
Did I miss something in this history?
I cannot see @pedrobaeza and @omar7r contributions to this module (though those are well know names :) )

https://github.com/OCA/department/commits/8.0/crm_department